src/deprecated/platformSpecificDeprecated.ios.js

```diff
@@ -232,7 +234,11 @@ function navigatorPush(navigator, params) {
     console.error('Navigator.push(params): params.screen is required');
     return;
   }
+  let previewViewID;
   const screenInstanceID = _.uniqueId('screenInstanceID');
+  if (params.previewView instanceof Component) {
+    previewViewID = findNodeHandle(params.previewView)
+  }
```

Comment:
I think it should have an `else` branch to treat `previewView` as `previewViewID` directly?

Reply:
Great idea, thanks!